sum function of two parameter

IgorConst opened the pull request:
Первая задача

Commit:
Added task Первая задача

jsru commented:
Спасибо за отправленный пулриквест, теперь @dosandk должен запустить тесты.

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

IgorConst commented:
Сделал у себя уже три коммита, запушил git push origin master, но не вижу тут своих изменений. Что я делаю не так?

IgorConst commented:
или на каждый push origin master нужно создавать свой пуллРеквест? чтобы тесты автоматом прочекались

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

IgorConst commented:
--Вижу, что у меня локально только мастер
dotNET@pasocon2 MINGW64 /d/js-sumost/js-20220513_sumost (master)
$ git branch
* master

--Вижу, что на апстриме новая ветка:
$ git fetch upstream
From https://github.com/js-tasks-ru/js-20220513_sumost
 * [new branch]      solutions  -> upstream/solutions

-- нагуглил как создать себе
$ git checkout --track origin/solutions
Switched to a new branch 'solutions'
branch 'solutions' set up to track 'origin/solutions'.

--сделал обновление всего
$ git fetch --all
Fetching origin
Fetching upstream

--внес корректировку в index.js чтобы были модификации
$ git add .

dotNET@pasocon2 MINGW64 /d/js-sumost/js-20220513_sumost (solutions)
$ git status
On branch solutions
Your branch is up to date with 'origin/solutions'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        new file:   01-intro/1-sum/index.js

--сделал коммит
dotNET@pasocon2 MINGW64 /d/js-sumost/js-20220513_sumost (solutions)
$ git commit -am "use branch solutions for commit"
[solutions 054aa75] use branch solutions for commit
 1 file changed, 20 insertions(+)
 create mode 100644 01-intro/1-sum/index.js

--сделал пуш origin solutions
dotNET@pasocon2 MINGW64 /d/js-sumost/js-20220513_sumost (solutions)
$ git push origin solutions
Enumerating objects: 6, done.
Counting objects: 100% (6/6), done.
Delta compression using up to 4 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (5/5), 604 bytes | 604.00 KiB/s, done.
Total 5 (delta 1), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
To https://github.com/IgorConst/js-20220513_sumost.git
   2632249..054aa75  solutions -> solutions

--попробовал тест запустить - не работает. Подумал что тест только с мастера заработают. Перешел в мастер
dotNET@pasocon2 MINGW64 /d/js-sumost/js-20220513_sumost (solutions)
$ npm test
No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In D:\js-sumost\js-20220513_sumost
  8 files checked.
  testMatch: **/__tests__/**/*.[jt]s?(x), **/?(*.)+(spec|test).[tj]s?(x) - 0 matches
  testPathIgnorePatterns: \\node_modules\\ - 8 matches
  testRegex:  - 0 matches
Pattern:  - 0 matches

dotNET@pasocon2 MINGW64 /d/js-sumost/js-20220513_sumost (solutions)
$ git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.

-- запушил в мастер - говорит нет изменений.
dotNET@pasocon2 MINGW64 /d/js-sumost/js-20220513_sumost (master)
$ git push origin master
Everything up-to-date

--Подскажите, где я неправильно делаю?
